### PR TITLE
smart turret default logic

### PR DESCRIPTION
**Pull request by 0xxlegolas**
- implemented smart turret default logic 
- added `characterId` as a additional parameter for view functions as `inProximity` and `aggression`
- normalized naming convention

### CODE REVIEW

File: mud-contracts/world/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,15 @@
...
 
+- smart turret feature implementation
+- update smart character creation (Breaking Change in interface)
+  - added corpId to `createCharacter` function
+  - added function to update corpId
+
+## 0.0.11
+
+### Patch Changes
+
```
Comment (tinkrtailor) [inline mud-contracts/world/CHANGELOG.md]: this probably should not be in here

**Review by tinkrtailor — APPROVED** [review]
one tiny comment, LGTM otherwise

**Review by tinkrtailor — APPROVED** [review]
one tiny comment, LGTM otherwise

File: mud-contracts/world/CHANGELOG.md (hunk 1 above)
Comment (0xxlegolas) [reply mud-contracts/world/CHANGELOG.md]: fixed

**Review by ledablack — APPROVED** [review]

**Pull request merged**